CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # gradle-nunit-plugin changelog
 ## 1.12
+### Changed
+* upgrade gradle-download-plugin to 3.2
+* upgrade net.researchgate.release to 2.6.0
+* built with gradle 4.2.1
```

Comment:
Will this still work with earliers Gradle versions?

Reply:
I have tested the the task can still be run on gradle 2.14 or 3.5 as well so it should be safe